setup/templates/steps/compose/01_config.html

```diff
@@ -0,0 +1,67 @@
+{% call macros.panel("info", "Step 2 - Initial configuration") %}
+<p>Before starting some variables must be set</p>
+
+<div class="form-group">
+  <label>Root path: </label>
+<!--   Validates path -->
+  <input class="form-control" type="text" name="root" value="/mailu" required pattern="^/[-_A-Za-z0-9]+(/[-_A-Za-z0-9]*)*">
+</div>
+
+<div class="form-group">
+  <label>Main mail domain</label>
```

Comment:
We should probably write some details about what the main mail domain is. Maybe even stop calling it that name: it is the server display name, maybe.